macros/edr/tests/on_run_end/handle_tests_results.sql

```diff
@@ -371,13 +368,14 @@
     {{ return(most_recent_anomaly_scores) }}
 {% endmacro %}
 
-{% macro get_elementary_test_table(database_name, schema_name, test_name, suffix) %}
-    {% set tests_schema_name = schema_name ~ elementary.get_config_var('tests_schema_name') %}
+{% macro get_elementary_test_table(test_relations, test_name, suffix) %}
     {% set test_table_name = elementary.table_name_with_suffix(test_name, suffix) %}
-    {% set test_table_relation = adapter.get_relation(database=database_name,
-                                                      schema=tests_schema_name,
-                                                      identifier=test_table_name) %}
-    {{ return(test_table_relation) }}
+    {% for test_relation in test_relations %}
```

Comment:
_Might_ need to optimize this. `test_relations` is an [`agate.Table`](https://agate.readthedocs.io/en/latest/api/table.html) object, it has `.find()` methods but I imagine it does roughly the same.